flowbased-computation/flowbased-computation-impl/src/test/java/com/farao_community/farao/flowbased_computation/impl/FlowbasedComputationImplTest.java

```diff
@@ -165,4 +197,39 @@ private double getCurativePtdf(FlowbasedComputationResult result, String conting
                 ))
             ));
     }
+
+    /* MOCKED ANYWAY...
```

Comment:
I realized that the tests were based on a mock sensi computation. So my flowbased modifications have no effect...